piet-common/src/cairo_back.rs

```diff
@@ -6,11 +6,47 @@ use cairo::{Context, Format, ImageSurface};
 
 use piet::{ErrorKind, ImageFormat};
 
+#[doc(hidden)]
 pub use piet_cairo::*;
 
 /// The `RenderContext` for the Cairo backend, which is selected.
 pub type Piet<'a> = CairoRenderContext<'a>;
 
+/// The associated brush type for this backend.
+///
+/// This type matches `RenderContext::Brush`
+pub type Brush = piet_cairo::Brush;
+
+/// The associated text factory for this backend.
+///
+/// This type matches `RenderContext::Text`
+pub type PietText<'a> = CairoText;
```

Comment:
Interesting, I didn't know you could introduce arbitrary lifetime annotations in an alias.